cloud-data-protection/src/features/userSlice.tsx

```diff
@@ -0,0 +1,21 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+export const userSlice = createSlice({
+    name: 'user',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
    name: "user",
```